Improve/move manage cost pg (#3491)
Qiskit/documentation: docs/guides/manage-cost.mdx

---

> title: Manage cost on paid plans

Author: ```suggestion
title: Manage cost
```

description: How to manage costs of running jobs on QPUs when using the Premium, Flex, or Pay-As-You-Go Plan on IBM Quantum Platform.

---

> # Manage cost on paid plans

Author: ```suggestion
# Manage cost
```

---
title: Manage cost on the Pay-As-You-Go Plan
description: How to manage costs of running jobs on QPUs when using the Pay-As-You-Go Plan on IBM Quantum Platform.

---

# Manage cost

> This guide describes how to set a cost limit on your Pay-As-You-Go Plan instance.  You can also manage cost by [minimizing a workload's quantum time](/docs/guides/minimize-time), and setting a [maximum execution time](/docs/guides/max-execution-time) on your jobs.

Author: Sure - and do we need to replace "jobs" across the whole platform to fix terminology? I don't understand the nuances between the two words, but it sounds like it's important?

    </Admonition>

<span id="cost-limit"></span>
## Set or change a cost limit

There are several ways to set or change a limit on how much is spent:

* **IBM Quantum Platform**: Open the [Instances](https://quantum.cloud.ibm.com/instances) table. Find the instance you want to limit, click the overflow menu at the end of the row, and then click **Edit details**. Set the total cost limit or the usage limit, then click **Save changes**.
   <Admonition type="note">
   You can also set the cost or usage limit when creating a new instance under the Pay-As-You-Go Plan. When you select the Pay-As-You-Go tile, a field appears with the label "Total cost limit (Optional)". You can enter a total cost limit in this field.
   </Admonition>

* **API**: [Set the instance limit in seconds.](/docs/api/qiskit-runtime-rest/tags/instances#tags__instances__operations__replace_instance_data)

   ```api
   PUT /v1/instances/configuration "instance_limit":<seconds>
   ```

* **IBM Cloud CLI**: For instructions to use the CLI, refer to [Getting started with the IBM Cloud CLI.](https://cloud.ibm.com/docs/cli?topic=cli-getting-started)

>    1. Log in to the CLI: `ibmcloud login --sso`.

Reviewer: Not specific to the changes happening, but it feels odd to include the basics of CLI usage in this article vs. just having the link to it. Basically, sharing the how to login and establish the session seems out of scope. I'd expect to just have the CLI command in this article. Not a major thing but feels like it makes maintenance a bit harder if we include this guidance in multiple places where the CLI is referenced.

> * **IBM Quantum Platform**: The Max cost limit is shown on the Standard tab of the [Instances & allocation tab.](https://quantum.cloud.ibm.com/instances)

Author: How is this tab labeled now?

> * **IBM Quantum Platform**: The Max cost limit is shown on the Standard tab of the [Instances & allocation tab.](https://quantum.cloud.ibm.com/instances)

Author: ```suggestion
* **IBM Quantum Platform**: The Max cost limit is shown on the Pay-As-You-Go tab of the [Instances & allocation](https://quantum.cloud.ibm.com/instances) tab.
```

## How to see what you’re being charged

> You or your account owner will receive a monthly invoice that provides details about your resource charges. You can check how much has been spent at any time on the [IBM Cloud Billing and usage page](https://cloud.ibm.com/billing).

Author: Is "or your account owner" true in this statement (also in line 158)? @fvarchon @Matt-Stypulkoski
